### ISSUE TITLE
Device shows as Unknown product even though manufacturer, product and product code are detected correctly

### ISSUE THREAD

**Issue by chaosjug** [zwave-js/zwave-js-ui#3401]
### Checklist

- [X]  I am not using Home Assistant. Or: a developer has told me to come here.
- [X] I have checked the troubleshooting section and my problem is not described there.
- [X] I have read the changelog and my problem is not mentioned there.

### Deploy method

Docker

### Z-Wave JS UI version

9.3.1.f26b97e

### ZwaveJS version

12.2.3

### Describe the bug

I'm just starting with z-wave so I have only two devices. An UZB  stick and FGR222.
In the control panel both show up as "Unknown manufacturer" / "Unknown product" but in both cases manufacturer, product and product code (the hex numbers) are correct and if I click on the link next to the ids the correct page on devices.zwave-js.io opens.
As a result I can't configure the devices.
Reinterviewing did not help. 
![image](https://github.com/zwave-js/zwave-js-ui/assets/3767486/65f00e46-717e-4cdb-88a7-ee215d753722)

### To Reproduce

When I first started, both devices showed up correctly. Then I moved the docker to another machine. (I use a folder not volumes, so I just copied the folder). After starting up again I reconfigured the device for the z-wave stick (now tcp (ser2net) instead of local device). And now both devices are "Unknown product". As I don't know how to get back, I don't know if I could reproduce it that way.

### Expected behavior

Devices should be detected correctly.

**Comment by robertsLando** [comment]
Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

**Comment by apella12** [comment]
Could also try to reinterview node (at least the device, not sure about controller)

**Comment by chaosjug** [comment]
Debug logging was already enabled so I had some files to choose from. I took the one from yesterday as zwave-js was restarted several times, I hope that gives more information than just the system running without much activity. Note that the host which provides the zwave stick with ser2net was offline for some time which caused the docker to restart several times.
[zwavejs_2023-11-02.log.gz](https://github.com/zwave-js/zwave-js-ui/files/13254464/zwavejs_2023-11-02.log.gz)

If I should do something specific while logging please tell me.
Regarding the reinterview: As stated above, I did that already several times with both with no effect.

**Comment by chaosjug** [comment]
I start the docker with a systemd service with this command:
docker run --name=%n -p 8091:8091 -p 3000:3000 \
  -v /opt/zwave:/usr/src/app/store \
  zwavejs/zwave-js-ui:9

Everything what's in /opt/zwave/ came from the other machine, where I did originally set things up. Where should the device configuration files be and how do they get there? Can I trigger a download somewhere and do I need to map another directory in addition to /usr/src/app/store to keep those files permanently?

**Comment by chaosjug** [comment]
That did the trick! Thanks a lot!
I moved the folder away so I could check the differences. A lot of files are missing in the old folder. From what I could reconstruct, I did not move that folder to the new host, as it is a hidden directory. What I don't understand is why it wasn't recreated correctly at first, but was recreated correctly now.
I'll close this issue for now. If I can reproduce the issue, I'll reopen or open new issue.
